src/data/weather.js

```diff
@@ -0,0 +1,41 @@
+const denverWeather = {
```

Comment:
I'm going to guess this is a stub for now? That's totally cool, but I might consider denoting it as such.